docs/OMS-Agent-for-Linux.md

```diff
@@ -671,12 +671,17 @@ An example section of the `omsagent.conf` configuration file `/etc/opt/microsoft
 </source>
 ```
 # Agent Logs
-The logs for the Operations Management Suite Agent for Linux can be found at: 
-`/var/opt/microsoft/omsagent/<workspace id>/log/`
-The logs for the omsconfig (agent configuration) program can be found at: 
-`/var/opt/microsoft/omsconfig/log/`
-Logs for the OMI and SCX components (which provide performance metrics data) can be found at:
-`/var/opt/omi/log/ and /var/opt/microsoft/scx/log`
+
+OMS Agent has a log directory for each workspace.  Other aspects of the framework have just one directory.  Both Agent configuration (omsconfig) and the SCX have just a single log file.
+Here are the directory paths of the four (note you need a workspace id GUID for the first one):
```

Comment:
To be consistent with the rest of the doc, call it "a workspace ID", if you mention this at all.